### PR TITLE
Update diagrams showing how microarray/RNA-seq work 

### LINKED ISSUE #255: Prettier/better diagrams showing how RNA-seq/microarray work
Based on @envest's request for moar worfklow graphics, I searched the internet for useful diagrams, and was only kind of successful: https://github.com/AlexsLemonade/refinebio-examples/pull/232#issuecomment-701477282

I found some placeholder figures that will work for now and illustrate the basic point of what we want in our rnaseq and microarray section headers, but they are not great looking, and also pretty low res. 

It would be good for us to make some nice workflow diagrams that explain the basic concepts behind the technologies that would be higher res and be more aesthetically pleasing. They should follow whatever color palette is used for #199 

The current, not great figures: 

https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/microarray-overview.jpg

https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/rnaseq-overview.png

### PR DESCRIPTION
**Pull request by cbethell**
## Purpose:
What issue(s) does your PR address?

This PR closes #255.

## Strategy
What was your strategy for this new or edited analysis?

Per the [original comment on #255](https://github.com/AlexsLemonade/refinebio-examples/issues/255#issue-712223762), I recreated the diagrams showing how microarray/RNA-seq work based on the figures currently in master:

[microarray diagram](https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/microarray-overview.jpg)
[RNA-seq diagram](https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/rnaseq-overview.png)

The intention here was to create higher resolution diagrams, and a more accurate single-color microarray diagram that corresponds with the data in refine.bio.

## Concerns/Questions for reviewers:
What things should reviewers look out for?

- Do the images in this PR satisfy the suggestions made on issue #255? cc: @cansavvy and @dvenprasad
- I converted the `microarray-overview.jpg` file to a PNG file (for consistency with the other images in the directory). I can change this file back to JPG if necessary.
- I added a sentence below each diagram as a brief description of what's going on in them. Please let me know if these descriptions are not clear or need more context.

### Content checks
* [x] All `{{BLANKS}}` have been replaced with the correct content.
* [x] [Sources are cited](https://github.com/AlexsLemonade/refinebio-examples/blob/master/CONTRIBUTING.md#citing-sources-in-text)
* [x] Seed is set (not applicable)

### Formatting Checks
* [x] Removed any manual numbering of sections.
* [x] Removed any instances of chunk naming.
* [x] Comments and documentation are up to date.
* [x] All links have been checked and are properly formatted.

### Add datasets to S3
* [x] Added [data and metadata files to S3](https://github.com/AlexsLemonade/refinebio-examples/blob/master/CONTRIBUTING.md#adding-datasets-to-the-S3-bucket).

### Docker/Snakemake rendering components
* [x] Added the `.html` [link to the navigation bar](https://github.com/AlexsLemonade/refinebio-examples/blob/master/CONTRIBUTING.md#add-new-analyses-to-the-navbar).
* [x] Any not yet added packages needed for this analysis have been added to the Dockerfile and it successfully builds.

### CODE REVIEW

**Comment by cansavvy** [comment]
The two main arrows in `components/figures/microarray-overview.png` look like their two different sizes or am I imagining this? 

Otherwise I think this PR should be ready to go.

**Comment by cansavvy** [comment]
Do we have the original files for these images noted somewhere in case we need to come back to edit them?

**Comment by cbethell** [comment]
> The two main arrows in `components/figures/microarray-overview.png` look like their two different sizes or am I imagining this?
> 
> Otherwise I think this PR should be ready to go.

I had the same thought, so I deleted it then made a duplicate of the first main arrow (still looked a little off to me but the measurements on google slides are the same so it may be our imagination?)

> Do we have the original files for these images noted somewhere in case we need to come back to edit them? Google drive or something?

Good point, I edited them on Google Slides so I'll put them on the shared Google drive.

**Comment by cansavvy** [comment]
> I had the same thought, so I deleted it then made a duplicate of the first main arrow (still looked a little off to me but the measurements on google slides are the same so it may be our imagination?)

If you copy and pasted and the arrows still look different, it is because we are imagining it. ¯\_(ツ)_/¯

File: 02-microarray/00-intro-to-microarray.Rmd [hunk 1]
```diff
@@ -32,12 +32,13 @@ Microarrays measure gene expression using chips filled with oligonucleotide prob
 After hybridization, the microarrays are scanned, and the fluorescence intensity for each probe is measured.
 The fluorescence intensity indicates the number of labeled fragments bound and therefore the relative quantity of the transcript the probe is designed for. 
 
-<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/microarray-overview.jpg" width=400>  
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/72ba39735440ea40051a3e753a71ae80f1094eac/components/figures/microarray-overview.png" width=400>
 
-[@microarray-video]
+[based on @microarray-video]
```
Comment (cansavvy) [inline 02-microarray/00-intro-to-microarray.Rmd:37]:
```suggestion
[based on diagram from @microarray-video]
```

¯\_(ツ)_/¯

File: 03-rnaseq/00-intro-to-rnaseq.Rmd [hunk 2]
```diff
@@ -36,10 +36,11 @@ This tutorial has example analyses [organized by technology](../01-getting-start
 
 RNA-sequencing measures gene expression by direct high-throughput sequencing methods after the RNA has been isolated from a sample. 
 
-<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/rnaseq-overview.png" width=400>  
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/72ba39735440ea40051a3e753a71ae80f1094eac/components/figures/rnaseq-overview.png" width=400>  
 
-[@rnaseq-blog]
+[based on @rnaseq-blog]
```
Comment (cansavvy) [inline 03-rnaseq/00-intro-to-rnaseq.Rmd:41]:
```suggestion
[based on diagram from @rnaseq-blog]
```

File: 02-microarray/00-intro-to-microarray.Rmd [hunk 3]
```diff
@@ -32,12 +32,13 @@ Microarrays measure gene expression using chips filled with oligonucleotide prob
 After hybridization, the microarrays are scanned, and the fluorescence intensity for each probe is measured.
 The fluorescence intensity indicates the number of labeled fragments bound and therefore the relative quantity of the transcript the probe is designed for. 
 
-<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/microarray-overview.jpg" width=400>  
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/72ba39735440ea40051a3e753a71ae80f1094eac/components/figures/microarray-overview.png" width=400>
```
Comment (cansavvy) [inline 02-microarray/00-intro-to-microarray.Rmd]:
```suggestion
<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/72ba39735440ea40051a3e753a71ae80f1094eac/components/figures/microarray-overview.png" width=600>
```

Now that they are pretty, we should make them bigger!! SO people can see the pretty work!

File: 03-rnaseq/00-intro-to-rnaseq.Rmd [hunk 4]
```diff
@@ -36,10 +36,11 @@ This tutorial has example analyses [organized by technology](../01-getting-start
 
 RNA-sequencing measures gene expression by direct high-throughput sequencing methods after the RNA has been isolated from a sample. 
 
-<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/rnaseq-overview.png" width=400>  
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/72ba39735440ea40051a3e753a71ae80f1094eac/components/figures/rnaseq-overview.png" width=400>
```
Comment (cansavvy) [inline 03-rnaseq/00-intro-to-rnaseq.Rmd]: This one seems fine as width 400 to me.

**Review by cansavvy — APPROVED** [review]
These are definitely prettier. Couple tiny suggestions, but then seems ready to me.

**Review by dvenprasad — APPROVED** [review]
👍

**Pull request merged**